examples/hosts/app-integration/external-data/src/model/containerCode.ts

```diff
@@ -62,11 +62,11 @@ export class TaskListContainerRuntimeFactory extends ModelContainerRuntimeFactor
         );
         // Register listener only once the model is fully loaded and ready
         runtime.on("signal", (message) => {
-            if (message.type === SignalType.ExternalDataChanged) {
-                // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        if (message?.content?.type === SignalType.ExternalDataChanged) {        // eslint-disable-next-line @typescript-eslint/no-floating-promises
                 taskList.importExternalData();
```

Comment:
Recommend adding a `.catch(console.error)` at least rather than floating the promise